CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to
 
 ### Added
 
+- functionality to download and save FewsNet data
+- unittests for downloading FewsNet data
```

Comment:
I don't think we need to add this to the changelog unless it's done in a version that is separate from the functionality